app/classifier/classifier.jsx

```diff
@@ -168,6 +168,8 @@ class Classifier extends React.Component {
     if (this.props.workflow.configuration.hide_classification_summaries && !this.subjectIsGravitySpyGoldStandard()) {
       if (!feedbackActive || (feedbackActive && !isThereFeedback(this.props.subject, this.props.workflow))) {
```

Comment:
I don't understand how this can ever be true, since it ORs `!feedbackActive` and `feedbackActive`.

Reply:
It should be true if feedback is disabled, OR if feedback is enabled but this subject doesn't have any feedback metadata (the latter is a nested expression)

Reply:
OK, so if feedback is enabled and the subject has feedback, then #4078 is triggered because nothing is submitted.